csp/middleware.py

```diff
@@ -29,6 +29,11 @@ class PolicyParts:
     nonce: str | None = None
 
 
+class FalseLazyObject(SimpleLazyObject):
+    def __bool__(self) -> bool:
+        return False
```

Comment:
Do you think a test just for this would be worthwhile?

Reply:
yes, see below

Reply:
I saw those. I meant more on its own... not wrapped up in the middleware. More a question if you think that covers it well enough or should there be separate unit tests to test these isolated. I'm fine with what's here in this PR.

Reply:
Ah. No, I think this is testing at the right level. I don't expect others to use this class. I could rename to `_FalseLazyObject` to make it clearer that it is private.